Add card locking / receipt policy post (#153)
hackclub/hcb-engr: content/posts/card-locking/post.mdx

import post from '@/components/Post.js'

export default post({

>   title: '💳 We\'ve updated card locking',

Reviewer: I'd name this something else since card locking isn't something most people already know about

>   title: '💳 Updates to how your cards are locked',

Reviewer 2: ```suggestion
  title: '📝 An update to HCB's receipt policy',
```

  slug: 'card-locking',
  category: 'improvement',

>   tags: ["cards"],

Reviewer 2: Is there also a `receipts` tag?

>   authors: ["manu", "gary", "samuel"],

Reviewer: +1

  date: new Date("2026-07-16"),
})

Starting August 11th, you will have 7 days to upload a receipt after each card charge settles. If 7 days pass and a transaction is still missing its receipt, you risk having your HCB cards locked. For most users, the impact is minimal: keep uploading your receipts on time, and you can spend on HCB just like you do today.

If your cards are locked, all of your HCB cards stop working for new purchases until you upload the missing receipt(s). Uploading unlocks them right away within seconds. Past transactions aren't affected, only new spending will be paused.

Receipts have always been required on HCB, but enforcing that by hand is slow and pulls time away from supporting all of our organizations. Handling it automatically keeps everyone compliant, protects the 501(c)(3) status your organization operates under, and frees up our team to give you faster, better support.

> See our [help center article](https://help.hcb.hackclub.com/en/articles/15410229-what-counts-as-a-valid-receipt-and-when-is-it-due#h_9852cf9992) for more information.

Reviewer 2: ```suggestion
We're updating HCB's receipt policy. Receipts are still required, just as they always have been. The IRS requires receipts to document how your nonprofit organization spends its money. What's changing is what happens when one is missing.
 
Starting August 11th, 2026, you will have 7 days to upload a receipt after each card charge settles. If 7 days pass and a transaction is still missing its receipt, you risk having your HCB cards locked. For most users, the impact is minimal: keep uploading your receipts on time, and you can spend on HCB just like you do today.
 
If your cards are locked, all of your HCB cards stop working for new purchases until you upload the missing receipt(s). Uploading unlocks them right away. Past transactions aren't affected; only new spending will be paused.
 
Receipts have always been required on HCB, but enforcing that by hand is slow and pulls time away from supporting all of our organizations. Handling it automatically keeps everyone compliant, protects the 501(c)(3) status your organization operates under, and frees up our team to give you faster, better support.
 
 
If you have any questions or concerns, please reach out to hcb@hackclub.com, and our team will be happy to help. See our [Help Center article](https://c.vialoops.com/CL0/https:%2F%2Fhelp.hcb.hackclub.com%2Fen%2Farticles%2F15410229-what-counts-as-a-valid-receipt-and-when-is-it-due%23h_9852cf9992/1/0100019fa9e15df6-5ca36d14-8ebc-4f8d-9d39-53a64dbf2e43-000000/VTB0A9WsjuqUNR57xM591v_DdCgEheaOAv7vqrG1_yg=452) for more information. Thank you for using HCB!
```

import post from '@/components/Post.js'

export default post({
  title: '📝 An update to HCB\'s receipt policy',
  slug: 'card-locking',
  category: 'improvement',
  tags: ["cards", "receipts"],
  authors: ["manu", "gary", "samuel"],
  date: new Date("2026-07-16"),
})

Starting August 11th, 2026, you will have 7 days to upload a receipt after each card charge settles. If 7 days pass and a transaction is still missing its receipt, you risk having your HCB cards locked. For most users, the impact is minimal: keep uploading your receipts on time, and you can spend on HCB just like you do today.

If your cards are locked, all of your HCB cards stop working for new purchases until you upload the missing receipt(s). Uploading unlocks them right away. Past transactions aren't affected; only new spending will be paused.

Receipts have always been required on HCB, but enforcing that by hand is slow and pulls time away from supporting all of our organizations. Handling it automatically keeps everyone compliant, protects the 501(c)(3) status your organization operates under, and frees up our team to give you faster, better support.
 
 

> If you have any questions or concerns, please reach out to hcb@hackclub.com, and our team will be happy to help. See our [Help Center article](https://c.vialoops.com/CL0/https:%2F%2Fhelp.hcb.hackclub.com%2Fen%2Farticles%2F15410229-what-counts-as-a-valid-receipt-and-when-is-it-due%23h_9852cf9992/1/0100019fa9e15df6-5ca36d14-8ebc-4f8d-9d39-53a64dbf2e43-000000/VTB0A9WsjuqUNR57xM591v_DdCgEheaOAv7vqrG1_yg=452) for more information. Thank you for using HCB!

Reviewer 2: ```suggestion
If you have any questions or concerns, please reach out to hcb@hackclub.com, and our team will be happy to help. See our [Help Center article](https://help.hcb.hackclub.com/en/articles/15410229-what-counts-as-a-valid-receipt-and-when-is-it-due#h_9852cf9992) for more information. Thank you for using HCB!
```

import post from '@/components/Post.js'

export default post({

>   title: '📝 An update to HCB's receipt policy',

Reviewer 2: ```suggestion
  title: '📝 An update to HCB\'s receipt policy',
```

Receipts have always been required on HCB, but enforcing that by hand is slow and pulls time away from supporting all of our organizations. Handling it automatically keeps everyone compliant, protects the 501(c)(3) status your organization operates under, and frees up our team to give you faster, better support.

We're updating HCB's receipt policy. Receipts are still required, just as they always have been. The IRS requires receipts to document how your nonprofit organization spends its money. What's changing is what happens when one is missing.
 
Starting August 11th, 2026, you will have 7 days to upload a receipt after each card charge settles. If 7 days pass and a transaction is still missing its receipt, you risk having your HCB cards locked. For most users, the impact is minimal: keep uploading your receipts on time, and you can spend on HCB just like you do today.
 
If your cards are locked, all of your HCB cards stop working for new purchases until you upload the missing receipt(s). Uploading unlocks them right away. Past transactions aren't affected; only new spending will be paused.
 

> Receipts have always been required on HCB, but enforcing that by hand is slow and pulls time away from supporting all of our organizations. Handling it automatically keeps everyone compliant, protects the 501(c)(3) status your organization operates under, and frees up our team to give you faster, better support.

Reviewer 2: ```suggestion
Starting August 11th, 2026, you will have 7 days to upload a receipt after each card charge settles. If 7 days pass and a transaction is still missing its receipt, you risk having your HCB cards locked. For most users, the impact is minimal: keep uploading your receipts on time, and you can spend on HCB just like you do today.

If your cards are locked, all of your HCB cards stop working for new purchases until you upload the missing receipt(s). Uploading unlocks them right away. Past transactions aren't affected; only new spending will be paused.

Receipts have always been required on HCB, but enforcing that by hand is slow and pulls time away from supporting all of our organizations. Handling it automatically keeps everyone compliant, protects the 501(c)(3) status your organization operates under, and frees up our team to give you faster, better support.
```
